tutorials/source-modeling/plot_source_space_SNR.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""
+.. _tut-mne-fixed-free:
```

Comment:
I suggest using a cross-reference target name that is more specific to what this tutorial *uniquely* covers. (e.g., include "snr" somewhere in the cross-ref name)

Reply:
done